src/pkg/packager/creator/normal_test.go

```diff
@@ -56,3 +58,44 @@ func TestDifferentialPackagePathSetCorrectly(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadPackageDefinition(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name      string
+		testDir   string
+		expectErr bool
+	}{
+		{
+			name:      "valid package definition",
+			testDir:   "valid",
+			expectErr: false,
+		},
+		{
+			name:      "invalid package definition",
+			testDir:   "invalid",
+			expectErr: true,
+		},
+	}
+
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+
+			src := layout.New(filepath.Join("testdata", tt.testDir))
+			pc := NewPackageCreator(types.ZarfCreateOptions{}, "")
+			pkg, _, err := pc.LoadPackageDefinition(src)
+
+			switch {
+			case tt.expectErr:
```

Comment:
Change this to a if statement which checks if expected error and the returns after check, becomes much cleaner.